tlm_teamd/teamdctl_mgr.cpp

```diff
@@ -136,6 +142,15 @@ bool TeamdCtlMgr::remove_lag(const std::string & lag_name)
     {
         SWSS_LOG_WARN("The LAG '%s' hasn't been added. Can't remove it", lag_name.c_str());
     }
+
+    // If this lag interface errored last time, clear it
+    if ((lag_name.compare(last_errored_lag_name) == 0) && (no_of_retry != 0))
```

Comment:
should we move this to get_dump()?

Reply:
We would need this check here in remove_lag()... as I mentioned in earlier comment -- that the lag could get cleared from the m_handlers here in this call to remote_lag() from update_interfaces(). Hence without this here, the no_retry count won't be reset by the logic even if we add it at line 196.